tests/gold_tests/redirect/redirect_post.test.py

```diff
@@ -63,7 +63,7 @@
 )
 
 tr = Test.AddTestRun()
-tr.Processes.Default.Command = 'curl -i http://127.0.0.1:{0}/redirect1'.format(ts.Variables.port)
+tr.Processes.Default.Command = 'touch largefile.txt && truncate largefile.txt -s 50M && curl -i http://127.0.0.1:{0}/redirect1 -F "filename=@./largefile.txt" && rm -f largefile.txt'.format(ts.Variables.port)
```

Comment:
We should add a `Condition.HasProgram` for `truncate`, or even better, write the file in python and avoid the dependency.

Reply:
+1 - MacOS doesn't have truncate installed be default:
```
09:59:20 mac:~$ truncate
-bash: truncate: command not found
```